blogposts/2022/release-3.37.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Sourcegraph 3.37 release'
+publishDate: 2022-02-22T10:00-07:00
+description: 'Sourcegraph 3.37 introduces performance improvements for Code Intelligence, sharing for Notebooks, and a new UI for creating search contexts.'
+tags: [blog, release]
+slug: 'release/3.37'
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.37/sourcegraph-3-37-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.37/sourcegraph-3-37-release.png
+changelogItems:
+  - description: 'Code in search results is now selectable (e.g. for copying). Just clicking on the code continues to open the corresponding file as it did before.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/30033
+    category: Search
+  - description: 'Query-based search contexts are now enabled by default as a beta feature, making it easier to scope queries for monorepos or large codebases.'
+    url: https://docs.sourcegraph.com/code_search/how-to/search_contexts#beta-query-based-search-contexts
+    category: Search
+  - description: 'Search Notebooks now support importing and exporting Markdown-formatted files.'
+    url: https://github.com/sourcegraph/sourcegraph/issues/28586
+    category: Search
+  - description: 'GraphQL API: The order of events returned by `MonitorTriggerEventConnection` has been reversed so newer events are returned first. The `after` parameter has been modified accordingly to return events older the one specified, to allow for pagination. TODO'
```

Comment:
```suggestion
  - description: 'GraphQL API: The order of events returned by `MonitorTriggerEventConnection` has been reversed so newer events are returned first. The `after` parameter has been modified accordingly to return events older than the one specified, to allow for pagination. TODO'
```
@iskyOS - this still says TODO.

Reply:
I haven't been able to find the corresponding PR (it said TODO in the changelog) — I've asked the engineer to provide it

Reply:
```suggestion
  - description: 'GraphQL API: The order of events returned by `MonitorTriggerEventConnection` has been reversed so newer events are returned first. The `after` parameter has been modified accordingly to return events older the one specified, to allow for pagination.'
```